editor/icons/ShowInFileSystem.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16" height="16" version="1.1" viewBox="0 0 16 16" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><path d="m1.5245 0.99323v2.3321h5.4722v4.6427h3.0505v4.6326h4.9578v-2.3321h-2.6225v-2.3005h2.6225v-2.3321h-5.6731v-2.3106h5.6731v-2.3321z" fill="#e0e0e0" stroke-width="1.1668"/><g transform="matrix(.81623 0 0 .81623 -3.294 2.7125)"><path d="m10.327 6.4248c-1.9131 0-4.3329 1.4574-5.2037 4.2684-0.038464 0.13447-0.038464 0.27704 0 0.41151 0.83648 2.9599 3.3333 4.2984 5.2037 4.2984 1.8705 0 4.3665-1.3386 5.2083-4.2834 0.03847-0.13447 0.03847-0.27704 0-0.41151-0.82302-2.8357-3.2973-4.2834-5.2083-4.2834zm0 1.4965c3.9904 0 3.9904 5.9856 0 5.9856-3.9904 0-3.9904-5.9856 0-5.9856zm0 1.4965c-1.9952 0-1.9952 2.9928 0 2.9928 1.9952 0 1.9952-2.9928 0-2.9928z" fill="#5fff97" stroke-width=".7482"/></g></svg>
```

Comment:
The SVG is not fully optimized. You can use SVG Cleaner for that.

Reply:
Also CC @MewPurPur 
It's a new icon, so you might want to take a look.

Reply:
Sure.

Reply:
```<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path d="M1 1v2.25h6v4.5h3v4.5h5V10h-2.75V7.75H15V5.5H9.25V3.25H15V1z" fill="#e0e0e0"/><path d="M5.04 8.04c-1.534 0-3.475 1.169-4.173 3.423a.6.6 0 0 0 0 .33c.67 2.374 2.673 3.447 4.173 3.447s3.502-1.073 4.177-3.435a.6.6 0 0 0 0-.33C8.557 9.201 6.572 8.04 5.04 8.04zm0 1.2a2.4 2.4 0 0 1 0 4.8 2.4 2.4 0 0 1 0-4.8zm0 1.2a1.2 1.2 0 0 0 0 2.4 1.2 1.2 0 0 0 0-2.4z" fill="#5fb2ff"/></svg>```

I changed the green to a blue, feel free to change the fill back.

Reply:
I'd say a co-authorship for the icon might be appropriate

Reply:
> I'd say a co-authorship for the icon might be appropriate

Is there something I need to write somewhere in regards to that?

Reply:
Yes, you can add `Co-authored-by: MewPurPur <mew.pur.pur@gmail.com>` at the end of your commit message's body. Make sure to skip one line after the main content.